From: Francois Cartegnie <fcvlcdev@free.fr>
Date: Fri, 14 Jun 2019 10:27:31 +0200
Subject: codec: subsvtt: unbreak rendering (fix #22425)

5ef3830f385bfa6a47e9088f9a0d13062cd61c3a regression

(cherry picked from commit 749b7b3729a5892d1a07036ef0974c17ddbe9e2e)
---
 modules/codec/webvtt/subsvtt.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/modules/codec/webvtt/subsvtt.c b/modules/codec/webvtt/subsvtt.c
index 2f79fb4..29d27d8 100644
--- a/modules/codec/webvtt/subsvtt.c
+++ b/modules/codec/webvtt/subsvtt.c
@@ -1808,11 +1808,7 @@ static void Render( decoder_t *p_dec, mtime_t i_start, mtime_t i_stop )
 
     GetTimedTags( p_sys->p_root->p_child, i_start, i_stop, &timedtags );
     if( timedtags.i_count == 0 )
-    {
-        vlc_array_clear( &timedtags );
-        return;
-    }
-    qsort( timedtags.pp_elems, timedtags.i_count, sizeof(*timedtags.pp_elems), timedtagsArrayCmp );
+        qsort( timedtags.pp_elems, timedtags.i_count, sizeof(*timedtags.pp_elems), timedtagsArrayCmp );
 
     mtime_t i_substart = i_start;
     for( size_t i=0; i<timedtags.i_count; i++ )
